### PR TITLE
[PRM-344] Make late appeal reason mandatory

### PR DESCRIPTION
**Pull request by AndrewRButcher**

### CODE REVIEW

**Comment by AndrewRButcher** [comment]
Screenshot of new error message
Screenshot 2021-06-16 at 10 25 42
Change requires update to ATs before merging

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
See #44 for revised error rendering

**Review by LewisMitchell — APPROVED** [review]

**Comment by LewisMitchell** [comment]
Waiting on AT changes

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
See ticket for changes.

**Comment by AndrewRButcher** [comment]
Screenshot 2021-06-16 at 13 36 28
Changes made based on UX feedback

File: conf/messages.cy [hunk 1]
```diff
@@ -188,5 +188,5 @@ appealConfirmation.feedbackLink.2 = (takes 30 seconds)
 # ----------------------------------------------------------
 makingALateAppeal.title = Making a late appeal
 makingALateAppeal.heading = Making a late appeal
-makingALateAppeal.h2 = Tell us why you did not appeal sooner
-makingALateAppeal.p1 = You usually need to appeal within 30 days of being told about a penalty. If you do not appeal within 30 days, we’ll use information you provide here to decide whether we can consider your late appeal.
\ No newline at end of file
+makingALateAppeal.p1 = You usually need to appeal within 30 days of being told about a penalty. If you do not appeal within 30 days, we’ll use information you provide here to decide whether we can consider your late appeal.
+makingALateAppeal.error.required = Explain why you did not appeal sooner
```
Comment (LewisMitchell) [inline conf/messages.cy:192]: Sorry to be a pain, but could you copy the makingALateAppeal.error.required key-value over from the English file?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: conf/messages.cy (hunk 1 above)
Comment (AndrewRButcher) [reply conf/messages.cy]: Done

**Review by LewisMitchell — APPROVED** [review]
Awesome, ready to merge when AT's are in.

**Pull request merged**